lib/components/viewers/route-viewer.js

```diff
@@ -1,13 +1,13 @@
-import coreUtils from '@opentripplanner/core-utils'
 import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import PropTypes from 'prop-types'
 import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
 import { FormattedMessage, injectIntl } from 'react-intl'
+import coreUtils from '@opentripplanner/core-utils'
+import PropTypes from 'prop-types'
 
```

Comment:
Sort imports

Reply:
Sorted in 
092b53f aside from React which I think needs to come first